static/bidder-info/rtbhouse.yaml

```diff
@@ -9,4 +9,4 @@ userSync:
   # rtbhouse supports user syncing, but requires configuration by the host. contact this
   # bidder directly at the email address in this file to ask about enabling user sync.
   supports:
-    - redirect
\ No newline at end of file
+    - redirect
```

Comment:
Please do not make changes to other adapters.

Reply:
I'm sorry. There was an accidental commit.